reference/forms/types/dateinterval.rst

```diff
@@ -128,6 +129,26 @@ your underlying object. Valid values are:
 The value that comes back from the form will also be normalized back into
 this format.
 
+labels
+~~~~~~
+
+.. versionadded:: 3.3
+   The ``labels`` option was introduced in Symfony 3.3.
+
+**type**: ``array`` **default**: (see below)
+
+The labels displayed for each of the elements of this type. The default values are::
+
+  'labels' => array(
+      'invert' => 'form.trans.invert',
```

Comment:
@HeahDude thanks for explaining the error. Hopefully I've fixed it in my other commit.